src/weapon.py

```diff
@@ -26,40 +24,34 @@ def __init__(self, input_button: VirtualButton, car: arcade.Sprite):
     def update(self):
         ...
 
-    def draw(self):
-        self.bullet_list.draw()
 
-
-class Beam(Weapon):
+class LaserBeam(Weapon):
     """
     stays on while button is pressed and moved with the ship
     """
 
     beam_projection: arcade.Sprite
+    beam_range: float
 
     def __init__(self, input_button: VirtualButton, car: arcade.Sprite):
         super().__init__(input_button, car)
-        self.beam_projection = arcade.SpriteSolidColor(1000, 5, arcade.color.RED)
+        self.beam_range = 500
+        self.beam_projection = arcade.SpriteSolidColor(self.beam_range, 5, arcade.color.RED)
+        self.beam_projection.velocity = self.beam_range/2
```

Comment:
You explained this to me on Discord.

It looks confusing, since `velocity` implies movement, but in this case there is no movement.

If the variable or attribute names are unclear, then I think we need to augment with code comments to explain.

Reply:
Added comment